packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -13,6 +13,7 @@ import { PresentationHierarchyNode, PresentationTreeNode } from "./Types";
 /** @beta */
 export interface HierarchyLevelFilteringOptions {
   hierarchyNode: HierarchyNode;
+  hierarchyLevelSizeLimit?: number | "unbounded";
```

Comment:
I don't feel like the limit belongs to an interface we call "filtering options". Maybe the interface (and the methods returning it) should be renamed to be more generic? E.g. `HierarchyLevelConfiguration` or something like that?

Reply:
Rename